tests/functional/adapter/snapshot_tests/seeds.py

```diff
@@ -0,0 +1,33 @@
+SEED_CSV = """
```

Comment:
nit: we've typically been using `fixtures.py`, maybe nest these files under a fixtures directory or add `_fixtures` as a suffix?